src/components/ProductDetail/Comment.tsx

```diff
@@ -2,7 +2,22 @@ import "./Comment.css";
 
 import getTimeElapsed from "../../utils/getTimeElapsed";
 
-function Comment({ comment }) {
+interface Writer {
+  image: string;
+  nickname: string;
+}
```

Comment:
API 명세를 봤을때 Writer라는 데이터 구조에 `id` 속성이 있는데 해당 코드에는 없네요. 
API 응답값이랑 동일하게 타입을 지정해준다고 생각하시면 편할 것 같습니다. 

```suggestion
interface Writer {
  image: string;
  nickname: string;
  id: number
}
```